feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.html

```diff
@@ -41,7 +41,9 @@
             {{ 'customerTicketing.category' | cxTranslate }}
           </span>
           <select class="form-control">
-            <option></option>
+            <option *ngFor="let category of ticketCategories">
```

Comment:
similarly to https://github.com/SAP/spartacus/pull/16142/files#r988362362

it doesn't do anything. It just 'displays' a dropdown, but the values aren't being passed to the APIs